scripts/add-local-ip-certs-to-docker.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Helper script to download the certificate and private key
+# from the http://local-ip.co service and install it
+# in medic-os in nginx. This script is safe to re-run
+# as many times as needed or if you need to refresh expired
+# certificates.
+#
+# After running this, you get a valid certificate with the format
+# of https://YOUR-IP-HERE.my.local-ip.co
+#
+# For example, if your localhost IP is 10.0.2.4, you would use this as
+# your URL for CHT:
+# https://10-0-2-4.my.local-ip.co
+
+if ! command -v docker&> /dev/null
```

Comment:
Cool, I didn't know you could do this. I'd have done some lame hack like `which docker` which isn't really correct. Thanks for the education!

Reply:
Yes!! I actually JUST learned about this when ~~manifested it out of thin air from my awesome brain~~ I stumbled upon [this SO post](https://stackoverflow.com/a/677212) this week.